src/pages/guides/api/eventsingress_api.md

```diff
@@ -13,6 +13,12 @@ title: Events Publishing API
    * create your own `Custom Events Provider`
    * create at least one `Event Metadata` associated with the above
 
+<InlineAlert variant="help" slots="header, text1" />
+
+Data Residency:
+
+Please note that all custom events are stored in US East region. 
+
```

Comment:
The richer version of [inline alert](https://github.com/adobe/aio-theme#richer-inlinealert) is not getting rendered properly. Do we need to update the [gatsby theme](https://github.com/AdobeDocs/adobe-io-events/pull/41) to get this working?

Reply:
For some reason, it does not even render in my local machine (like it did in your screenshot)

![npm run dev output](https://user-images.githubusercontent.com/6910192/181739285-b9c4511b-2bc0-42ba-ae3a-43febbfd33c2.png)

Reply:
Updating gatsby theme didn't help either. 

Ya, I also got this screenshot when I used `...text1, text2` in `slots`.

Reply:
Here's how it looks after my last commit  https://github.com/AdobeDocs/adobe-io-events/pull/51/commits/d6b63571c5877ccf469ec1c610025a77c4a39de3 - 

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/5928555/181744486-6d7d91ea-5284-41d8-8c9f-90faa912150a.png">